src/pal/inc/pal.h

```diff
@@ -3303,6 +3303,56 @@ BitScanForward64(
     return bRet;
 }
 
+// Define BitScanReverse64 and BitScanReverse
+// Per MSDN, BitScanReverse64 will search the mask data from MSB to LSB for a set bit.
+// If one is found, its bit position is returned in the out PDWORD argument and 1 is returned.
+// Otherwise, 0 is returned.
+//
+// On GCC, the equivalent function is __builtin_clzll. It returns 1+index of the most
+// significant set bit, or 0 if if mask is zero.
```

Comment:
It returns the number of leading 0-bits. If the argument is 0, the result is undefined.

Reply:
For reference: https://gcc.gnu.org/onlinedocs/gcc/Other-Builtins.html